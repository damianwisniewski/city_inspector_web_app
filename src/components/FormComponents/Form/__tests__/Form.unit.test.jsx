import React from 'react'
import { configure, shallow, ShallowWrapper } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'

import Form from '../Form'

configure({ adapter: new Adapter() })

describe('<Form />', () => {
	/**
	 * @type {ShallowWrapper}
	 */
	let FormElement
	const FormContent = <input />
	const id = 'mockId'

	test('should render correctly Form', () => {
		FormElement = shallow(<Form id={id}>{FormContent}</Form>)
		expect(toJson(FormElement)).toMatchSnapshot()
	})
})
