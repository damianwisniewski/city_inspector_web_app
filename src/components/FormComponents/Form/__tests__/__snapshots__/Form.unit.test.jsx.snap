// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should render correctly Form 1`] = `
<form
  id="mockId"
>
  <input />
</form>
`;
